bases/br_cgu_servidores_remuneracao/microdados_militar_ativo/table_config.yaml

```diff
@@ -0,0 +1,705 @@
+# Se essa pessoa é você, preencha abaixo com suas informações.
+data_cleaned_by:
+    name: Maurício Fagundes
+    email: mauricio.fagundes@gmail.com
+    github_user: mfagundes
+    ckan_user: mfagundes
+    website:
+
+# Se houve passos de tratamento, limpeza e manipulação de dados, descreva-os aqui.
+data_cleaning_description:
+  - Download dos dados da CGU com tratamento para compatibilização de valores com o formato da Base dos Dados
+
+# Url do código de limpeza dos dados.
+data_cleaning_code_url:
+  - /code_
+
+# Organização que ajudou institucionalmente na disponibilização dos dados.
+partner_organization:
+    name:
+    organization_id:
+
+# Url dos dados originais no GCP Storage.
+raw_files_url:
+
+# Url dos arquivos auxiliares no GCP Storage.
+auxiliary_files_url:
+
+# Url da tabela de arquitetura no GCP Storage.
+architecture_url:
+
+source_bucket_name: basedosdados-dev
+
+project_id_prod: basedosdados-dev
+
+project_id_staging: basedosdados-dev
+
+# Liste as colunas da tabela que representam partições.
+# Não esqueça de deletar essas colunas nas tabelas .csv na hora de subir para o BigQuery.
+# Isso poupará muito tempo e dinheiro às pessoas utilizando essa tabela.
+# Se não houver partições, não modifique abaixo.
+partitions:
+    - ano
+    - mes
+
+# Quais são as colunas? Certifique-se de escrever uma boa descrição, as pessoas vão gostar
+# para saber sobre o que é a coluna.
+# Adicionar todas as colunas manualmente pode ser bastante cansativo, por isso, quando
+# inicializando este arquivo de configuração, você pode apontar a função para uma amostra de dados que
+# preencherá automaticamente as colunas.
+# Algumas colunas existirão apenas na tabela final, você as construirá em `publish.sql`.
+# Para esses, defina is_in_staging como False.
+# Além disso, você deve adicionar as colunas de partição aqui e definir is_partition como True.
+columns:
+    - name: ano
+      bigquery_type: int64
+      description: Ano
+      temporal_coverage:
+          - 2013-01(1)2022-08
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id: br_bd_diretorios_data_tempo
+          table_id: ano
+          column_name: ano
+      measurement_unit: year
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition: true
+    - name: mes
+      bigquery_type: int64
+      description: Mês
+      temporal_coverage:
+          - 2013-01(1)2022-08
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id: br_bd_diretorios_data_tempo
+          table_id: mes
+          column_name: mes
+      measurement_unit: month
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition: true
+    - name: id_servidor_portal
+      bigquery_type: string
+      description: 'ID do servidor no Portal da Transparência '
+      temporal_coverage:
+          - 2013-01(1)2022-08
```

Comment:
substituir temporal_coverage por (1)

Reply:
Acho que esse veio preenchido da tabela de arquitetura. Se for, melhor alterar lá, correto?